### ISSUE TITLE
LocalDev: WordPress site codebase changes are reflected on the local site only after restart

### ISSUE THREAD

**Issue by gabe-connolly** [pantheon-systems/localdev-issues#72]
This is very similar in behavior to #71 , but in a WordPress environment rather than a Drupal site.  When I make changes to my local codebase, short of a full restart of the local project my changes are not appearing.  What is interesting is that when I first started local development on this project, this was not an issue.  It was only after enabling CDN Emulation and Redis Cache via the Localdev UI that this issue started up.  I have tried toggling both those cache settings off in Localdev, and event went so far as to try and disable cache in my lando file and then run a `lando rebuild`, but this has not resolved the issue.

```
cache: false
edge: false
```

Local environment:
Mac OS 10.15.5
Docker 2.3.0.3
LocalDev 0.10.1

**Comment by jveens** [comment]
@gabe-connolly Have you found a way around this issue in the meantime? I'm experiencing something similar, and it's just become too frustrating for me to keep trying to work with this app. Going to look at using `lando` without the app interface instead.

**Comment by gabe-connolly** [comment]
@jveens since this issue only popped up for me after I had enabled CDN Emulation and Redis Cache, I tried completely nuking my local environment and starting fresh.  With a fresh clone of the repo and a new instance of the lando project, changes seemed to be working as expected again.  This does mean that doing reliable testing with caching turned on is still going to be an issue, but for my immediate development needs this at least lets me see my changes without doing a cache clear each time.
